src/Compilers/CSharp/Portable/CodeGen/EmitExpression.cs

```diff
@@ -274,6 +274,10 @@ private void EmitExpressionCore(BoundExpression expression, bool used)
                     EmitPseudoVariableValue((BoundPseudoVariable)expression, used);
                     break;
 
+                case BoundKind.Void:
```

Comment:
Right, my mistake.

---
In reply to: [64932294](https://github.com/dotnet/roslyn/pull/11594#discussion_r64932294) [](ancestors = 64932294)